AdminWebsite/AdminWebsite/ClientApp/src/app/booking/judicial-office-holders/search-for-judicial-member/search-for-judicial-member.component.ts

```diff
@@ -5,6 +5,7 @@ import { JudiciaryPerson } from 'src/app/services/clients/api-client';
 import { debounceTime, tap } from 'rxjs';
 import { JudicialMemberDto } from '../models/add-judicial-member.model';
 import { Constants } from '../../../common/constants';
+import { HoursType } from '../../../common/model/hours-type';
```

Comment:
think this can be removed, doesn't appear to be used

Reply:
Ah missed that one, cheers!